doc/man7/provider-base.pod

```diff
@@ -405,6 +406,25 @@ The number of bits of security offered by keys in this group. The number of bits
 should be comparable with the ones given in table 2 and 3 of the NIST SP800-57
 document.
 
+=item "tls-group-is-kem" (B<OSSL_CAPABILITY_TLS_GROUP_IS_KEM>) <unsigned integer>
+
+Boolean flag to describe is the group should be used in KEX mode (0, default) or
+in KEM mode (1).
+
+This parameter is optional: if not specified, KEX mode is assumed as the default
+mode for the group.
+
+In KEX mode, in a typical Diffie-Hellman fashion, both sides execute I<keygen>
+then I<derive> against the peer public key.
+
+In KEM mode, the client executes I<keygen> and sends its public key, the server
```

Comment:
Needs to support the provider functions described in provider-kem.pod.....oh...wait...no such file :-(

Reply:
Partially fixed in upcoming commit: I am referencing the yet to be created `provider-kem(7)` page, and I will open a separate issue (or PR if I find the time) to fix the issue.

I believe it is better to treat the addition of that manpage separately from this PR, as that is a requirement for beta1 or the final release, while this PR adds a new feature on which the OTC hold label is pending.